rastervision/backend/torch_utils/chip_classification/data.py

```diff
@@ -74,13 +105,40 @@ def build_databunch(data_dir, img_sz, batch_sz, class_names, augmentors):
     valid_ds = AlbumentationDataset(
         ImageFolder(valid_dir, classes=class_names))
 
+    if rare_classes != []:
+        train_sample_weights = calculate_oversampling_weights(
+            train_ds, rare_classes, desired_prob)
+
+        def get_class_with_max_count(imageFolder):
+            count_per_class = {}
+            for class_idx in imageFolder.class_to_idx.values():
+                count_per_class[class_idx] = 0
+            for (sample_path, class_index) in imageFolder.samples:
+                count_per_class[class_index] += 1
+            largest_class_idx = max(count_per_class, key=count_per_class.get)
+            return count_per_class[largest_class_idx]
+
+        num_train_samples = len(
+            train_ds.classes) * get_class_with_max_count(train_ds)
```

Comment:
I don't understand why you don't just use the `len(train_ds)` as the `num_train_samples`. The way it's set right now doesn't seem correct, but maybe I'm missing something.

Reply:
I sent you a long version explanation via mail, but the short version is that I will apply the method as you explained here